massa-bootstrap/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-speed-limit = { git = "https://github.com/adrien-zinger/async-speed-limit", rev = "36d79e0", features = ["default", "tokio"] }
```

Comment:
From our last discussion I understood that the original repo contains all the necessary features. I'm surprised to see a fork being used here.